acai-ios/CollectionViewCells/DetailViewCollectionViewCells/HeaderImageCollectionViewCell.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  HeaderViewCollectionViewCell.swift
+//  acai-ios
+//
+//  Created by Artesia Ko on 3/7/19.
+//  Copyright © 2019 Cornell AppDev. All rights reserved.
+//
+
+import UIKit
+
+class HeaderImageCollectionViewCell: UICollectionViewCell {
+    
+    weak var delegate: dismissMenuItemViewController?
+    
+    // MARK: View vars
+    var dismissButton: UIButton!
+    var imageView: UIImageView!
+    
+    // MARK: View initialization
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        
+        imageView = UIImageView()
+        imageView.translatesAutoresizingMaskIntoConstraints = false
+        imageView.backgroundColor = .white
+        imageView.image = UIImage()
+        imageView.contentMode = .scaleAspectFit
+        imageView.clipsToBounds = true
+        imageView.layer.masksToBounds = true
+        self.addSubview(imageView)
+        imageView.snp.makeConstraints { (make) in
+            make.center.equalToSuperview()
+            make.width.equalToSuperview()
+            make.height.equalToSuperview()
+        }
+        
+        dismissButton = UIButton(type: .system)
+        dismissButton.translatesAutoresizingMaskIntoConstraints = false
+        dismissButton.backgroundColor = .clear
+        dismissButton.setImage(UIImage(named: "x_icon"), for: .normal)
```

Comment:
can you make `x_icon` a constant at the top of the class? i.e. `let dismissButtonImageName = "x_icon"`